src/main/java/se/michaelthelin/spotify/SpotifyApi.java

```diff
@@ -953,6 +953,20 @@ public CheckUsersSavedShowsRequest.Builder checkUsersSavedShows(String... ids) {
       .ids(concat(ids, ','));
   }
 
+  /**
+   * Check if one or more episodes is already saved in the current Spotify user's 'Your Episodes' library.
+   *
+   * @param ids The episode IDs to check for in the user's 'Your Episodes' library. Maximum: 50 IDs.
+   * @return A {@link CheckUsersSavedEpisodesRequest.Builder}.
+   * @see <a href="https://developer.spotify.com/documentation/web-api/concepts/spotify-uris-ids">Spotify: URLs &amp; IDs</a>
+   * @apiNote This endpoint is in <b>beta</b> and could change without warning.
```

Comment:
Hmm, `javadoc` generation fails with `error: unknown tag: apiNote`. Do you know what I can do, @hwup?

Reply:
try changing `maven-javadoc-plugin` to 
```text
<source>9</source>
<target>9</target>

Reply:
Weird, didn't work either. I made those comments normal comments. `v8.2.0` is on its way.